.github/workflows/scripts/check_bundle_size.js

```diff
@@ -0,0 +1,67 @@
+module.exports = {
+  checkBaseBundleSize: () => {
+    const fs = require('fs');
+    const jsonString = fs.readFileSync('.github/analyze/bundles.json');
+    const data = JSON.parse(jsonString);
+    const pagesToCheck = [
+      '/',
+      '/cli/start/install',
+      '/lib/auth/getting-started/q/platform/[platform]',
+      '/start',
+      '/cli'
+    ];
+    const baseBundleSizes = [];
+    data.pages.filter((page) => {
+      if (pagesToCheck.includes(page.label)) {
+        baseBundleSizes.push({
+          page: page.label,
+          gzipSize: page.gzipSize,
+          parsedSize: page.parsedSize,
+          statSize: page.statSize,
+          totalGzipSize: page.totalGzipSize,
+          totalParsedSize: page.totalParsedSize,
+          totalStatSize: page.totalStatSize
+        });
+      }
+    });
+    return baseBundleSizes;
+  },
+
+  checkHeadBundleSize: () => {
+    const fs = require('fs');
+    const jsonString = fs.readFileSync('.github/analyze/bundles.json');
+    const data = JSON.parse(jsonString);
+    const pagesToCheck = [
+      '/',
+      '/cli/start/install',
+      '/lib/auth/getting-started/q/platform/[platform]',
+      '/start',
+      '/cli'
+    ];
+    const headBundleSizes = [];
+    data.pages.filter((page) => {
+      if (pagesToCheck.includes(page.label)) {
+        headBundleSizes.push({
+          page: page.label,
+          gzipSize: page.gzipSize,
+          parsedSize: page.parsedSize,
+          statSize: page.statSize,
+          totalGzipSize: page.totalGzipSize,
+          totalParsedSize: page.totalParsedSize,
+          totalStatSize: page.totalStatSize
+        });
+      }
+    });
+    return headBundleSizes;
+  },
+
+  compareBundles: (baseBundles, headBundles) => {
+    for (let i = 0; i < baseBundles.length; i++) {
+      if (baseBundles[i] !== headBundles[i]) {
+        if (baseBundles[i].parsedSize * 1.05 < headBundles[i].parsedSize) {
+          return false;
```

Comment:
should this be `return true` it looks like the final steps condition `if: ${{ steps.compare-bundles.outputs.result }}` is looking for a value of `true` to block the PR or is it just any non null value?

Reply:
It's looking for any non-null value, but `true` is better. Changing

Reply:
Fixed this logic to be able to console.log any problem pages.